pootle/apps/pootle_language/forms.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django import forms
+
+from .models import Language
+
+
+class LanguageSpecialCharsForm(forms.ModelForm):
+
+    class Meta:
+        model = Language
+        fields = ('specialchars',)
+        labels = {
+            'specialchars': Language._meta.get_field('specialchars').help_text,
```

Comment:
Please let's not incur in such hackish ways to reverse the positioning of elements in form fields — form's layout can be easily modified in the template.